beacon/validator/src/test/java/tech/pegasys/teku/validator/coordinator/BlockFactoryDenebTest.java

```diff
@@ -109,8 +109,6 @@ void unblindSignedBlock_shouldUnblindBeaconBlock() {
     assertThat(unblindedBlockContainer).isInstanceOf(SignedBlockContents.class);
     assertThat(unblindedBlockContainer.isBlinded()).isFalse();
     assertThat(unblindedBlockContainer.getSignedBlock()).isEqualTo(unblindedBeaconBlock);
-    assertThat(unblindedBlockContainer.getSignedBlobSidecars())
-        .hasValueSatisfying(blobSidecars -> assertThat(blobSidecars).isEmpty());
```

Comment:
Shouldn't we leave a TODO to check blobs and kzg proofs once added?

Reply:
Yes. Good point. Added.